lib/magma/validation_object.rb

```diff
@@ -24,8 +24,8 @@ def options
     def match
     end
 
-    def as_json
-      @options.to_json
+    def to_hash
+      @options&.to_hash
     end
```

Comment:
The usual behavior of #to_json on generic_object is to call as_json.to_json - so in line 65 of `attribute.rb` you could write `validation: validation_object`. Later, when the retrieve endpoint calls to_json on the payload, to_json will call `attribute[:validation].as_json.to_json` - so here you could write:

    def as_json
      @options
    end

`@options` should already be a Hash and will respond correctly to `Hash#to_json` when called. The result will be the same as you've written but this method seems to be the idiomatic intent of `#as_json`.